Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -413,7 +413,7 @@ Note that for country code filtering, troll country codes should be prepended wi
 
     <!-- Main About group -->
     <string name="settings_about_license_description">Tap to see license</string>
-    <string name="settings_about_licenses">Open Source Licenses</string>
```

Comment:
This isn't an instance where hyphenation is needed.

Reply:
Alright then